### DISCUSSION TITLE
unable  to simulate the GATE

### DISCUSSION THREAD

**Discussion by santoshgate** [OpenGATE/opengate#489] [General]
messaaage: [Core-0] Initialization of geometry
WARNING: In /usr/local/include/ITK-6.0/itkImageSeriesReader.hxx, line 478
ImageSeriesReader (0x55dec5f3fc50): Non uniform sampling or missing slices detected,  maximum nonuniformity:4.50579

GateVImageVolume.cc (l.739):  I find R=-1024 in the image, while range stop at -1025

### WARNING ### setSkipEqualMaterials at false !! The Geant4 method is not safe since the release 9.5 - Need to be fixed

**Comment by tbaudier** [comment]
Hello @santoshgate

Can you explain and describe what are you doing please?
How did obtain this error?

I f you need help to run your fist simulation, you can find more information in the doc: https://opengate-python.readthedocs.io/en/latest/

**Reply by santoshgate** [reply]
doing for Internal dosimetry simulation with DICOM images and TIA dicom,
when simulate error come

**Reply by santoshgate** [reply]
santosh@DESKTOP-MLM4J4G:~/Gate/build$ ./Gate /home/santosh/Gate_simulations/patient4_simulation/macros/patient4.mac
[G4]
[G4] **************************************************************
[G4]  Geant4 version Name: geant4-11-02-patch-02 [MT]   (21-June-2024)
[G4]                        Copyright : Geant4 Collaboration
[G4]                       References : NIM A 506 (2003), 250-303
[G4]                                  : IEEE-TNS 53 (2006), 270-278
[G4]                                  : NIM A 835 (2016), 186-225
[G4]                              WWW : http://geant4.org/
[G4] **************************************************************
[G4]
[Core-0] Initialization of geometry
[Core-0] Initialization of physics
[Core-0] Initialization of actors
[Core-0]
[Core-0] *******************************************************
[Core-0]  GATE version 9.4 (2024)
[Core-0]  Copyright : OpenGATE Collaboration
[Core-0]  Reference : Phys. Med. Biol. 49(19) 4543-4561     2004
[Core-0]  Reference : Phys. Med. Biol. 56(4)  881-901       2011
[Core-0]  Reference : Med. Phys.       41(6)  1-14          2014
[Core-0]  Reference : Phys. Med. Biol. 66(10) 1-23          2021
[Core-0]  Reference : Frontiers in Physics, 12              2024
[Core-0]  http://www.opengatecollaboration.org
[Core-0] *******************************************************
[Core-0]
[Core-0] You are using Geant4 version 11.2.2
[Core-0] Starting macro /home/santosh/Gate_simulations/patient4_simulation/macros/patient4.mac
[Physic-0] The following Geant4's physic-list is enabled :emstandard


List of Enabled processes:


   ===  Process: IonIonisation  ===
Particle: GenericIon
Particle: alpha
Particle: deuteron
Particle: triton
Particle: He3



List of initialized processes:

  * GenericIon
    - msc
    - ionIoni
    - Radioactivation
  * He3
    - msc
    - ionIoni
  * alpha
    - msc
    - ionIoni
  * anti_He3
    - msc
    - hIoni
  * anti_alpha
    - msc
    - hIoni
  * anti_deuteron
    - msc
    - hIoni
  * anti_omega-
    - msc
    - hIoni
  * anti_proton
    - msc
    - hIoni
    - hBrems
    - hPairProd
    - CoulombScat
  * anti_sigma+
    - msc
    - hIoni
  * anti_sigma-
    - msc
    - hIoni
  * anti_triton
    - msc
    - hIoni
  * anti_xi-
    - msc
    - hIoni
  * deuteron
    - msc
    - hIoni
  * e+
    - msc
    - eIoni
    - eBrem
    - annihil
    - CoulombScat
  * e-
    - msc
    - eIoni
    - eBrem
    - CoulombScat
  * gamma
    - GammaGeneralProc
  * kaon+
    - msc
    - hIoni
    - hBrems
    - hPairProd
    - CoulombScat
  * kaon-
    - msc
    - hIoni
    - hBrems
    - hPairProd
    - CoulombScat
  * mu+
    - msc
    - muIoni
    - muBrems
    - muPairProd
    - CoulombScat
  * mu-
    - msc
    - muIoni
    - muBrems
    - muPairProd
    - CoulombScat
  * omega-
    - msc
    - hIoni
  * pi+
    - msc
    - hIoni
    - hBrems
    - hPairProd
    - CoulombScat
  * pi-
    - msc
    - hIoni
    - hBrems
    - hPairProd
    - CoulombScat
  * proton
    - msc
    - hIoni
    - hBrems
    - hPairProd
    - CoulombScat
  * sigma+
    - msc
    - hIoni
  * sigma-
    - msc
    - hIoni
  * tau+
    - msc
    - hIoni
  * tau-
    - msc
    - hIoni
  * triton
    - msc
    - hIoni
    - Radioactivation
  * xi-
    - msc
    - hIoni

[Core-0] Initialization of geometry
WARNING: In /usr/local/include/ITK-6.0/itkImageSeriesReader.hxx, line 478
ImageSeriesReader (0x556a5e731930): Non uniform sampling or missing slices detected,  maximum nonuniformity:4.50579

GateVImageVolume.cc (l.739):  I find R=-1024 in the image, while range stop at -1025

### WARNING ### setSkipEqualMaterials at false !! The Geant4 method is not safe since the release 9.5 - Need to be f

**Comment by tbaudier** [comment]
Your problem is related to Gate 9 (c++ version) and not opengate (python version).
Please, open an issue in that repository:
https://github.com/OpenGATE/Gate
